src/relayclient/AccountManager.ts

```diff
@@ -34,7 +45,7 @@ export default class AccountManager {
     const keypair = {
       privateKey: a.privKey,
       // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
remove disable lint..

Reply:
I don't want to

Reply:
then its better to move it to be beginning of the file, to ignore all errors...
seriously: if you refactored a line to use a function, then the `eslint-disable` should be moved with it.